web/src/emoji_picker.ts

```diff
@@ -714,14 +797,14 @@ function handle_status_emoji_clicked(emoji_name) {
     user_status_ui.toggle_clear_message_button();
 }
 
-function handle_composition_emoji_clicked(emoji_name) {
+function handle_composition_emoji_clicked(emoji_name: string): void {
     hide_emoji_popover();
     const emoji_text = ":" + emoji_name + ":";
     // The following check will return false if emoji was not selected in
     // message edit form.
     if (edit_message_id !== null) {
-        const $edit_message_textarea = $(
-            `#edit_form_${CSS.escape(edit_message_id)} .message_edit_content`,
+        const $edit_message_textarea = $<HTMLTextAreaElement>(
+            `#edit_form_${CSS.escape(edit_message_id.toString())} .message_edit_content`,
```

Comment:
TypeScript doesn’t parse selectors; make sure that any type assertion you make is verified at runtime.

```suggestion
        const $edit_message_textarea = $<HTMLTextAreaElement>(
            `#edit_form_${CSS.escape(edit_message_id.toString())} textarea.message_edit_content`,
```

Reply:
Done. Thanks.